aten/src/ATen/native/TensorIterator.h

```diff
@@ -86,11 +86,15 @@ struct CAFFE2_API OperandInfo {
   /// Stride after broadcasting. The stride is in bytes, not number of elements.
   DimVector stride_bytes;
 
-  /// The original tensor operand. Note that the strides, data pointer, and
+  /// The tensor operand. Note that the strides, data pointer, and
   /// other attributes may differ due to dimension reordering and
   /// coalescing.
   Tensor tensor;
 
+  // Buffer the original tensor operand in cases when an output is modified
+  // (e.g. if dtype is changed)
+  Tensor original_buffer;
```

Comment:
the name is a little confusing, because there is no such thing as an original buffer.  Honestly, original_tensor is probably clearer, but if you want original_tensor_as_buffer or something would work.

Reply:
renamed to original_tensor